### DISCUSSION TITLE
Is there a way to use sysparm_query for date ranges with -Filter?

### DISCUSSION THREAD

**Discussion by CanisVY** [Snow-Shell/servicenow-powershell#250] [Q&A] [answered]
Good afternoon and thank you for your development work with this module!

With the ServiceNow REST API explorer, it is possible to utilize a timeframe query with the sysparm_query field. 

Example being: 

`sysparm_query      sys_created_onBETWEENjavascript:gs.dateGenerate('2023-07-01','00:00:00')@javascript:gs.dateGenerate('2023-07-08','23:59:59')
`
which generates
`sys_created_onBETWEENjavascript%3Ags.dateGenerate('2023-07-01'%2C'00%3A00%3A00')%40javascript%3Ags.dateGenerate('2023-07-08'%2C'23%3A59%3A59')
`

Is it possible to do something like this with generated date ranges in SN?

**Comment by gdbarron** [comment, marked as answer]
If you are referring to `Get-ServiceNowRecord`, I believe what you are looking for is this help example
![image](https://github.com/Snow-Shell/servicenow-powershell/assets/11862024/a82d4a17-778f-410f-9b8f-8c484d63ec1c)

**Reply by CanisVY** [reply]
Thank you very much for the syntax help! This is even easier.
